### ISSUE TITLE
On Manjaro, can't locate vstring.h

### ISSUE THREAD

**Issue by andy5995** [cade-vs/vfu#14]
Hey @cade-vs ! Nice to see you've made a lot of progess. :)

I'm trying to build 4.21 on Manjaro, with gcc 11.1.0, but running into

```
../../vslib/eval.cpp:11:10: fatal error: vstring.h: No such file or directory
   11 | #include <vstring.h>
```
Looks like I've got it, but in some weird directories

```
$ locate vstring.h
/home/andy/src/Geany/geany/ctags/main/vstring.h
/home/andy/src/audacity-minsrc-2.4.2/lib-src/ffmpeg/libavutil/avstring.h
/usr/include/c++/11.1.0/ext/vstring.h
/usr/include/libavutil/avstring.h
```

I'm gonna try to get more info and maybe solve this. If you have any ideas though, please let me know.

Btw, I saw vfu is in the [AUR](https://aur.archlinux.org/packages/vfu-utf/)... I flagged it out of date so maybe the maintainer will fix it up with the latest release soon.

**Comment by andy5995** [comment]
Using `CXXFLAGS=-I/usr/include/c++/11.1.0/ext ../configure` gets rid of that error, but then I get a lot of other errors. I've attached the output and my config.log

[config.log](https://github.com/cade-vs/vfu/files/6804871/config.log)
[vfu-build.log](https://github.com/cade-vs/vfu/files/6804872/vfu-build.log)

**Comment by andy5995** [comment]
I see now that vstring.h is included with the vfu distribution.

When I don't use the configure script, and just run 'make' in the source directory, I get a build. :)

**Comment by cade-vs** [comment]
hello Andy!

thank you for this email(s) :)
I'm sorry for this delayed reply.

configure part of VFU release is third party.
I'm not sure how well it works.
plain 'make' should do it.

I'll check your last email to see if I can fix the configure.

thanks again and contact me anytime with whatever notes, problems, ideas you may have!

btw, I have new, up-to-date page for VFU 
(I managed to fix my site, which was few years overdue):

http://cade.noxrun.com/projects/vfu/

have a good week!
Vladi.
--


On Tue, 13 Jul 2021 12:32:05 -0700
Andy Alt ***@***.***> wrote:

> I see now that vstring.h is included with the vfu distribution.
> 
> When I don't use the configure script, and just run 'make' in the source directory, I get a build. :)
> 
> 
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/cade-vs/vfu/issues/14#issuecomment-879345328


-- 
Vladi Belperchinov-Shabanski  
***@***.***> ***@***.***> http://cade.datamax.bg
pgp/gpg key 6F35B214 @ http://pgp.mit.edu

**Comment by andy5995** [comment]
Your new website looks pretty cool.

I updated [vfu in the AUR repo](https://aur.archlinux.org/packages/vfu/)

Minor feedback about the site, but sometimes there are 2 slashes in the URLs (e.g. `http://cade.noxrun.com//projects/` ... seems to originate from the links on the left: "Home | Projects"

I've moved away from autotools the last couple months. I use @mesonbuild now for most of my projects now.

For building vfu, `make` is working fine for me. :)
